tools/ARIAtools/ARIAProduct.py

```diff
@@ -433,12 +433,21 @@ def __mappingVersion__(self, fname, version):
                 '/science/grids/imagingGeometry/azimuthAngle'
             ]
             if version.lower()=='1c':
+                # get weather model name
+                meta = gdal.Info(fname)
+                for i in meta.split():
+                    if '/science/grids/corrections/external/troposphere/' in i:
+                        model_name = i.split('/')[-3]
```

Comment:
@sssangha Can you decribe the behaviour if two different models are in the product. e.g. product 1 has model A/B and product 2 has model B

Reply:
Yeah [this part](https://github.com/aria-tools/ARIA-tools/blob/sss_refsec/tools/ARIAtools/ARIAProduct.py#L447-#L449) of the code is intended to assign the weather models as separate keys for this very reason. I.e. as separate keys, they will be sorted as separately within the dictionary.

But it would be a good idea to confirm this is working as I'd intended. @bbuzz31, could you perhaps make a dummy-duplicate layer of HRES in the test files you've provided?

Reply:
I tested this with 4 products: all 4 had HRES, 2 also had GMAO. 
The HRES outputs are written correctly, and no GMAO outputs are calculated.